native/src/boot/magiskboot.hpp

```diff
@@ -2,17 +2,20 @@
 
 #include <sys/types.h>
 
-#define HEADER_FILE     "header"
-#define KERNEL_FILE     "kernel"
-#define RAMDISK_FILE    "ramdisk.cpio"
-#define SECOND_FILE     "second"
-#define EXTRA_FILE      "extra"
-#define KER_DTB_FILE    "kernel_dtb"
-#define RECV_DTBO_FILE  "recovery_dtbo"
-#define DTB_FILE        "dtb"
-#define NEW_BOOT        "new-boot.img"
+#define HEADER_FILE         "header"
+#define KERNEL_FILE         "kernel"
+#define SECOND_FILE         "second"
+#define EXTRA_FILE          "extra"
+#define KER_DTB_FILE        "kernel_dtb"
+#define RECV_DTBO_FILE      "recovery_dtbo"
+#define DTB_FILE            "dtb"
+#define RAMDISK_FILE        "ramdisk.cpio"
+#define VENDOR_RAMDISK_FILE "vendor_ramdisk_%.*s.cpio"
+#define RAMDISK_TABLE_FILE  "ramdisk_table"
```

Comment:
What is this for? You added this but never used it.

Reply:
Forgot to remove it. At first i also duped the raw table, but that doesn't make much sense so I removed it.